test/common/include/alpaka/test/event/EventHostManualTrigger.hpp

```diff
@@ -978,7 +978,7 @@ namespace alpaka
                             0x01010101u,
                             CU_STREAM_WAIT_VALUE_GEQ)));
 #else
-                    // workaround for missing cuStreamWaitValue32 in HIP(HCC)
+                    // workaround for missing cuStreamWaitValue32 in HIP
                     std::uint32_t hmem = 0;
```

Comment:
how bad is the workaround impact here? I know AMD have had a ticket for this, but still seems to be not in HIP, so we could bump this issue.

Reply:
IMO it is not nice but only part of the testing where the performance will not be so negative effected.